CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # OverReact Changelog
 
+## [4.10.0](https://github.com/Workiva/over_react/compare/4.9.1...4.10.0)
+- [#829] Add improved prop forwarding method
+- [#809] Update json_annotation & serializable in example app
+- [#826] Allow test_html_builder 3
+- [#827] Raise minimums
```

Comment:
I don't think these are notable enough to call out individually in the changelog
```suggestion
- [#809], [#826], [#827] Example app dependency updates
```